src/applications/edu-benefits/0993/containers/SinglePageFormApp.jsx

```diff
@@ -38,18 +34,6 @@ class FormApp extends React.Component {
       }
     }
 
-    // Show nav only if we're not on the intro, form-saved, error, or confirmation page
```

Comment:
This page is essentially the us-forms-system's FormPage component with the FormNav component and related logic removed.